cocos2d/CCViewPortNode.h

```diff
@@ -0,0 +1,68 @@
+//
```

Comment:
This is nitpicky but the filename is CCViewPortNode.h while the class is CCViewportNode. I prefer the latter over the former though either works for me so long as the file and class are consistent.